dev-docs/bidders/prebidServer.md

```diff
@@ -22,12 +22,12 @@ Bid params are sourced from the adapter configurations set for client side. Thes
 To enable prebid server, set the following configuration. 
 
 ```
-pbjs.setS2SConfig({
-    accountId : '12345',
-    enabled : true,
-    bidders : ['appnexus','audienceNetwork', 'rubicon'], 
-    timeout : 1000, 
-    cookieSet : true
+pbjs.setConfig({
+    s2sConfig: {
+        accountId : '12345',
+        bidders : ['appnexus','audienceNetwork', 'rubicon'], 
```

Comment:
I think this block needs the k-v `enabled: true` according to the [Prebid Server bidder params](http://prebid.org/dev-docs/bidders.html#prebidServer)

Reply:
The `enabled` attribute is included under the `defaultVendor` k-v.

Reply:
👍